### PR TITLE
Try to fix make.bat for Windows 11

### PR DESCRIPTION
**Pull request by Julfried**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :bug: Bug fix          |
|    | :sparkles: New feature |
|    | :hammer: Refactoring   |
|    | :scroll: Docs          |

## Description
Follow Up to this: [10064](https://github.com/pylint-dev/pylint/pull/10064)

I noticed that my changes to make.bat do not work for my Windows 11 Laptop. So I fixed it in here

### CODE REVIEW

File: doc/make.bat [hunk 1]
```diff
@@ -11,184 +11,69 @@ if NOT "%PAPER%" == "" (
...
+	goto end
+)
+
+
+REM Help section
+:help
+echo. Please use `make ^<target^>` where ^<target^> is one of:
+echo.
+echo.  install-dependencies   to install required documentation dependencies
+echo.  html                   to make standalone HTML files
```
Comment (DanielNoord) [inline doc/make.bat:58]: I think we should only document the options we actually support

Comment (Julfried) [reply doc/make.bat:58]: Is all the other stuff even used somewhere? How about removing it completly?

Comment (DanielNoord) [reply doc/make.bat:58]: We removed it from the other make file as well!

**Review by DanielNoord — APPROVED** [review]
Thanks again!

**Pull request merged**